EssentialApp/EssentialAppTests/Helpers/ListViewController+TestHelpers.swift

```diff
@@ -93,3 +93,20 @@ extension ListViewController {
 
 	private var feedImagesSection: Int { 0 }
 }
+
+extension ListViewController {
+	func numberOfRenderedImageCommentViews() -> Int {
+		numberOfRows(in: imageCommentsSection)
+	}
+
+	func imageCommentView(at row: Int) -> UITableViewCell? {
+		cell(row: row, section: imageCommentsSection)
+	}
+
+	func commentMessage(at row: Int) -> String {
+		let imageCommentCell = imageCommentView(at: row) as! ImageCommentCell
+		return imageCommentCell.messageText!
+	}
```

Comment:
This method can return optional instead of force-unwrapping with `!` (which can crash the app):

```suggestion
	func commentMessage(at row: Int) -> String? {
		let cell = imageCommentView(at: row) as? ImageCommentCell
		return cell?.messageText
	}
```

This is better because if the message is `nil`, the test will fail. But with `!`, the test suite would crash and stop running.